Xamarin.MacDev/IPhoneDeviceType.cs

```diff
@@ -46,6 +46,22 @@ public enum AppleDeviceFamily
 		IPad   = 2,
 		TV     = 3,
 		Watch  = 4,
+		IPod   = 5,
+	}
+
+	public static class AppleDeviceFamilyExtensions
+	{
+		public static string GetDeviceClass (this AppleDeviceFamily family)
+		{
+			switch (family) {
+				case AppleDeviceFamily.TV: return "tvOS";
+				case AppleDeviceFamily.Watch: return "watch";
+				case AppleDeviceFamily.IPhone: return "iphone";
+				case AppleDeviceFamily.IPad: return "ipad";
+				case AppleDeviceFamily.IPod: return "ipod";
+				default: return string.Empty;
```

Comment:
Similar to `ToDeviceType` you might want to throw on default.

`throw new ArgumentOutOfRangeException (string.Format ("Unknown device family: {0}", family));`